src/coreclr/src/jit/codegenarm64.cpp

```diff
@@ -1925,21 +1925,37 @@ void CodeGen::genCodeForStoreLclVar(GenTreeLclVar* tree)
 {
     GenTree* data = tree->gtOp1;
 
-    // var = call, where call returns a multi-reg return value
-    // case is handled separately.
-    if (data->gtSkipReloadOrCopy()->IsMultiRegCall())
+    // Multi-reg nodes are handled separately.
```

Comment:
> IsMultiReg is only for GenTreeLclVarCommon, IsMultiRegNode and IsMultiRegLclVar are for all tree, is it correct?

Actually, `isMultiReg` is only for `GenTreeLclVar` (it must be `GT_LCL_VAR` not other variants).

> Maybe rename so:
GenTree has IsMultiRegNode, IsMultiRegLclVar(virtual)
GenTreeLclVarCommon has IsMultiRegLclVar.

I don't think we really have much of a "standard" for this, but since we don't really support virtual methods on `GenTree`, I believe we generally keep the names distinct.

 I'll rename `tree` to `lclNode` (I think that `lclVar` is confusing because one might expect it to be a `LclVarDsc*`. I'll make the same change to the version in codegenxarch.cpp.